src/main/java/com/twcable/jackalope/impl/jcr/SessionImpl.java

```diff
@@ -423,4 +446,63 @@ boolean isNew(ItemImpl item) {
     boolean isModified(ItemImpl item) {
         return changedItems.contains(item.getPath());
     }
+
+
+    private static class MyNamespaceRegistry implements NamespaceRegistry {
+        private final Map<String, String> prefixToUri = new HashMap<>();
+
+
+        public MyNamespaceRegistry() {
+            prefixToUri.put(Name.NS_JCR_PREFIX, Name.NS_JCR_URI);
+            prefixToUri.put(Name.NS_MIX_PREFIX, Name.NS_MIX_URI);
+            prefixToUri.put(Name.NS_NT_PREFIX, Name.NS_NT_URI);
+            prefixToUri.put(Name.NS_REP_PREFIX, Name.NS_REP_URI);
```

Comment:
I see that there are also `NS_SV_PREFIX`, `NS_XML_PREFIX` and `NS_XMLNS_PREFIX` in `org.apache.jackrabbit.spi.Name`. Should we add them to the defaults too?